app/Channel.php

```diff
@@ -18,7 +18,13 @@ class Channel extends Model
      *
      * @var array
      */
-    protected $fillable = ['channel_name', 'channel_description', 'subscription_count'];
+    protected $fillable = 
```

Comment:
let it be the way it was before.

`protected $fillable = ['channel_name', 'channel_description', 'subscription_count'];`